addons/account/models/chart_template.py

```diff
@@ -628,8 +626,8 @@ def should_delay(created_models, yet_to_be_created_models, model, field_name, fi
                     'values': deref_values(record_vals, self.env[model]),
                     'noupdate': True,
                 })
-            loaded_records[model] = self.with_context(lang='en_US').env[model]._load_records(all_records_vals, update=ignore_duplicates)
-        return loaded_records
+            created_records[model] = self.with_context(lang='en_US').env[model]._load_records(all_records_vals)
```

Comment:
I prefer the name which can reflect the stored data. Here it is not `created_records` but `loaded_records`. So that people won't abusively use the method. But since it is for account, I will leave the decision to your team.